ha/core/event_analyzer/parser/parser.py

```diff
@@ -137,7 +137,7 @@ def parse_event(self, msg: str) -> HealthEvent:
             }
             # To be removed after HA starts populating IEM messages
```

Comment:
Can this be removed now ?

Reply:
No

Reply:
Understood. This is for the IEM framework from foundation team